pkg/formatters/rawFormatter_test.go

```diff
@@ -114,3 +117,44 @@ func TestRawFormatterNoRunEndtimeReturnsBlankEndtimeFieldAndNoDuration(t *testin
 
 	assert.Equal(t, expectedFormattedOutput, actualFormattedOutput)
 }
+
+func TestRawFormatterOnlyLostTestReturnsBlankOutput(t *testing.T) {
```

Comment:
Can we please change these test function names so that they are a bit more consistent with other tests across galasa